content/getting-started/connect-the-pim-4x/step-2.md

```diff
@@ -0,0 +1,120 @@
+# _Step 2_ | Cook your connector
+
+In this step, there are 2 sub-steps.
+
+## Find your connector on our marketplace
+
+We have good news for you. Maybe the connector you have in mind already exists!
+
+Indeed, we have a marketplace on which our dear ecosystem publishes its own tools and connectors.
+
+So, we definitely recommend you to take a look at it and try to find a connector that can suit your use case. It will save you a lot of time!
+
+1. Navigate to the [marketplace](https://marketplace.akeneo.com/extensions/add-ons?edition=all&version=all&api_use=1&sort=date) website.  
+The results are already filtered to show you the API-based connectors only.
+1. In this list of connectors, try to find the one that can fit your use case best.
+
+You found what you were looking for? Great! You can now install your connector and go directly to the [third step of this tutorial](/getting-started/connect-the-pim-4x/step-3.html).
+
+Couldn't find any connector that suits you?  
+It is the case for our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.  
+Alright! So let's build it ourselves!
+
+## Build your very own connector
+
+In the case you can't find an already cooked connector on our marketplace, you will have to develop it. 
+
+Below are the macro steps to successfully go through this development.
+1. Have your connection use case in mind
+2. Find out what are the third-party connection possibilities
+3. Choose your integration frequency
+4. Explore our API reference
+5. Think about your architecture
+6. Ochestrate your global connector
+
+Those steps are detailed below, illustrated by our tutotrial example.
+
+### 1- Have your connection use case in mind
+
+We know it can seem a bit obvious but it really is the first step for a successful connection between the PIM and your third-party. You need to have at least the global scenario of your connection in mind.
+
+#### In this tutorial
+_You want to connect your ERP. Each day, new products are created into your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```

Comment:
```suggestion
_You want to connect your ERP. Each day, new products are created in your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```